packages/plugin-text/src/plugins/katex/editor.tsx

```diff
@@ -320,7 +320,7 @@ export const DefaultEditorComponent: React.FunctionComponent<
             {!isList(orderedListNode)(editor) &&
             !isList(unorderedListNode)(editor) ? (
               <InlineCheckbox
-                label="Inline"
+                label="Eingebettet"
```

Comment:
I think this is not really better... Maybe switch the checkbox the other way around then use the label "Eigene Zeile"?

Reply:
done